### PR TITLE
LPS-111049 Update dependencies

### PR DESCRIPTION
**Pull request by inacionery**

### CODE REVIEW

**Comment by inacionery** [comment]
ci:test:relevant

**Comment by inacionery** [comment]
ci:test:sf

File: modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl [hunk 1]
```diff
@@ -1689,11 +1691,46 @@ public abstract class Base${schemaName}ResourceTestCase {
...
+
+			if (graphQLField != null) {
+				Class<?> type = field.getType();
+
+				if (type.isArray()) {
+					type = type.getComponentType();
+				}
+
+				List<GraphQLField> childrenGraphQLFields = getGraphQLFields(
+					ReflectionUtil.getDeclaredFields(type));
```
Comment (tinatian) [inline modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl:1733]: @inacionery 
You should skip the checking for primary types and jdk types to make it fast.

File: modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl [hunk 2]
```diff
@@ -1689,11 +1691,46 @@ public abstract class Base${schemaName}ResourceTestCase {
...
+		for (Field field : fields) {
+			com.liferay.portal.vulcan.graphql.annotation.GraphQLField
+				graphQLField = field.getAnnotation(
+					com.liferay.portal.vulcan.graphql.annotation.GraphQLField.
+						class);
+
+			if (graphQLField != null) {
+				Class<?> type = field.getType();
+
+				if (type.isArray()) {
```
Comment (tinatian) [inline modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl:1728]: @inacionery I am not sure why you only care about array here, what about other collection types like List, Set and so on? 

Tina.

**Comment by tinatian** [comment]
@inacionery I am closing this cause there are conflicts, please rebase and resend if needed.

Thanks.
Tina.

File: modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl (hunk 2 above)
Comment (inacionery) [reply modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl:1728]: The generator uses only array

File: modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl (hunk 1 above)
Comment (inacionery) [reply modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl:1733]: got it, thanks!

**Pull request closed without merging**